packs/_source/classfeatures/paladin/paladin-features/lay-on-hands.json

```
@@ -93,29 +93,100 @@
             "choice": false,
             "special": ""
           },
-          "prompt": true,
+          "prompt": false,
           "override": false
         },
         "healing": {
           "number": null,
           "denomination": null,
           "bonus": "",
-          "types": [],
+          "types": [
+            "healing"
+          ],
           "custom": {
-            "enabled": false,
-            "formula": ""
+            "enabled": true,
+            "formula": "1"
           },
           "scaling": {
             "mode": "whole",
             "number": null,
-            "formula": ""
+            "formula": "1"
           }
         },
         "uses": {
           "spent": 0,
           "recovery": []
         },
         "sort": 0
+      },
+      "dnd5eactivity001": {
+        "_id": "dnd5eactivity001",
+        "type": "utility",
+        "name": "Cure Disease and Poison",
+        "activation": {
+          "type": "action",
+          "value": 1,
+          "condition": "Target cannot be Undead or a Construct.",
+          "override": false
+        },
+        "consumption": {
+          "targets": [
+            {
+              "type": "itemUses",
+              "target": "",
+              "value": "5",
+              "scaling": {
+                "mode": "amount",
+                "formula": "5"
+              }
+            }
+          ],
+          "scaling": {
+            "allowed": true,
+            "max": ""
```

Scaling max here also.

```suggestion
            "max": "floor(@item.uses.value / 5)"
```